.gitignore

```diff
@@ -29,3 +29,7 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+
+# Ignore mac specific files
+.DS_Store
```

Comment:
Follow up on comment from other PR, we should have a linter to standardize space at EOF